python/src/aiconfig/editor/client/src/components/ParametersRenderer.tsx

```diff
@@ -196,53 +196,56 @@ export default memo(function ParametersRenderer(props: {
-          color="dimmed"
-          size="sm"
-          p="xs"
-          style={{ display: "block", margin: "0 auto", textAlign: "right" }}
-        >
-          Use parameters in your prompt or system prompt with {"{{parameter}}"}
-        </Text>
-      )}
-      <Stack>
-        {parameters.map((parameter, i) => {
-          return (
-            <ParameterInput
-              onUpdateParameter={({ parameterName, parameterValue }) => {
-                setParameters((prev) => {
-                  const newParameters = [...prev];
-                  const currentElement = newParameters[i];
-                  currentElement.parameterName = parameterName;
-                  currentElement.parameterValue = parameterValue ?? "";
-
-                  onUpdateParameters(
-                    parametersArrayToJSONObject(newParameters)
-                  );
-
-                  return newParameters;
-                });
-              }}
-              removeParameter={(parameterName) =>
-                removeParameter(parameter.key, parameterName)
-              }
-              initialItemValue={{
-                parameterName: parameter.parameterName,
-                parameterValue: parameter.parameterValue,
-              }}
-              key={parameter.key}
-            />
-          );
-        })}
-      </Stack>
+    <div>
+      <div
+        style={{
+          maxHeight: props.maxHeight,
+          overflow: "auto",
+          width: "100%",
+        }}
+      >
+        {props.customDescription ?? (
+          <Text
+            color="dimmed"
+            size="sm"
+            p="xs"
+            style={{ display: "block", margin: "0 auto", textAlign: "right" }}
+          >
+            Use parameters in your prompt or system prompt with{" "}
+            {"{{parameter}}"}
+          </Text>
+        )}
+        <Stack>
+          {parameters.map((parameter, i) => {
+            return (
+              <ParameterInput
+                onUpdateParameter={({ parameterName, parameterValue }) => {
+                  setParameters((prev) => {
+                    const newParameters = [...prev];
+                    const currentElement = newParameters[i];
+                    currentElement.parameterName = parameterName;
+                    currentElement.parameterValue = parameterValue ?? "";
+
+                    onUpdateParameters(
+                      parametersArrayToJSONObject(newParameters)
+                    );
+
+                    return newParameters;
+                  });
+                }}
+                removeParameter={(parameterName) =>
+                  removeParameter(parameter.key, parameterName)
+                }
+                initialItemValue={{
+                  parameterName: parameter.parameterName,
+                  parameterValue: parameter.parameterValue,
+                }}
+                key={parameter.key}
+              />
+            );
+          })}
+        </Stack>
+      </div>
```

Comment:
what does the extra div do? it looks like an intermediate div between the stack and the AddParameter section